common/protos-js/.npmignore

```diff
@@ -4,4 +4,5 @@ node_modules/
 package-lock.json
 Makefile
 .gitignore
-.npmignore
\ No newline at end of file
+.npmignore
+build-protos.sh
```

Comment:
Why ignore this file?

Reply:
While publishing the npm module, build-script is not required (it will still be checked in, its not gitignore)